specs/commandline/sign.md

```diff
@@ -40,6 +41,34 @@ Flags:
   -m,  --user-metadata strings    {key}={value} pairs that are added to the signature payload
 ```
 
+## Use OCI image manifest to store signatures
+
+By default, Notation uses [OCI artifact manifest][oci-artifact-manifest] to store signatures in the registries. For backward compatibility, Notation supports using `OCI image manifest` to store the signature in the registries that implement partial of the [OCI Image specification v1.1][oci-image-spec]. Use flag `--image-spec v1.1-image` to force Notation store the signatures using OCI image manifest.
```

Comment:
I feel like we should also mention that `Notation will not automatically convert between OCI artifact manifest and OCI image manifest on any occasion`.

Reply:
IMO we shouldn't add that because thats implicit from explicitly using a flag. Also, when we do automatic fallback, we will need to delete it and which we might forget

Reply:
Keep the wording not changed for now. We can add the restriction in the future if needed.